mentoring/programs/shadow-roles.md

```diff
@@ -34,7 +34,7 @@ community by its contributors. Promoting the work and helping fill the needs of
 force behind the Kubernetes Upstream Marketing Team. The goal of this work is to improve the overall sustainability of 
```

Comment:
There are several mentions in this section of the "Kubernetes Upstream Marketing" team and the section is called "Upstream Marketing". We should rename all "Upstream Marketing" to "Contributor Comms"

Reply:
renamed